package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.7.0",
   "description": "A virtual DOM library with focus on simplicity, modularity, powerful features and performance.",
   "main": "snabbdom.js",
+  "module": "es/snabbdom.js",
```

Comment:
`"jsnext:main": "es/snabbdom.js",` should also be used, some tools are still using that one.